### ISSUE TITLE
Recipient username case ignored 

### ISSUE THREAD

**Issue by sylph1o** [tijder/SmsMatrix#46]
First of all, thank you for this awesome application, which I discovered only recently.

My problem is that the Matrix ID to which I want to send the SMS contains uppercase letters (e.g. `@myMatrixID:myserver.com`), and SmsMatrix seems to convert it into lowercase. When I connect to the bot account, I can see that all the messages have been sent to that same wrong Matrix ID (e.g. `@mymatrixid:myserver.com`). Thankfully, that wrong Matrix ID seems not to exist.

I know that it is a bad idea to have uppercase letters in my Matrix ID, but there is no easy way for me to change that. I hope that fixing this SmsMatrix bug will be simple.

**Comment by tijder** [comment]
Thanks for you're feedback. 

I didn't knew are is a difference in uppercase usernames. On the [Riot](https://riot.im/app/#/register) site it isn't possible to create a account with uppercase's in it. In the Android code I don't convert the username to lowercase. 
Just to be sure, their is no typo in the username?

**Comment by swedneck** [comment]
I definitely recall reading that the spec explicitly forbids using anything but lowercase in IDs, but that synapse/riot still supports it for.. reasons..

**Comment by tijder** [comment]
@sylph1o can other people invite you in the android app? I think the bug is in the Android SDK and not in the SmsMatrix app.

**Comment by sylph1o** [comment]
Wow, you answered so fast ! I was hoping to close this silly issue before anyone witnessed my stupidity.

Matrix ID are indeed sensitive to case. I did not know that, and discovered it by not receiving some invitations. I did not even want uppercase letters in my ID; it must have been a typing mistake which I had thought unimportant at the time. It's too late to change now ='(.

Back to the issue I opened too quickly: my tests were flawed, and the mistake was mine. I initially wrote the wrong, lowercase Matrix ID, then spotted and identified the problem, changed the information, observed the same problem and opened that issue. Since then, I have restarted the service and now everything work as it should. I must have forgotten to save the Matrix ID change, or some such mistake.

Sorry for wasting your time. This app is great and works for me =).
